src/test/java/org/junit/runners/CustomBlockJUnit4ClassRunnerTest.java

```diff
@@ -0,0 +1,148 @@
+/*
+ * Copyright 2002-2015 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
This has been replaced with a license header compliant with the Eclipse Foundation.